modules/data/src/main/java/io/fluo/webindex/data/Copy.java

```diff
@@ -31,6 +29,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.fluo.webindex.core.DataConfig;
+import io.fluo.webindex.data.spark.IndexEnv;
```

Comment:
We have a style guide for imports:

https://github.com/fluo-io/fluo/wiki/Contributing